README.md

```diff
@@ -32,21 +32,17 @@ you use the default version.
 
 Simply running a container with the image will launch a Perl 6 REPL:
 
-    $ docker run -it rakudo-star
+    $ docker run --rm -it rakudo-star
```

Comment:
Probably note this above as `Simply running a (temporary) container` or somesuch...

Reply:
Changed.